packages/app-store/qr_code/zod.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+
+import { eventTypeAppCardZod } from "../eventTypeAppCardZod";
+
+export const appDataSchema = eventTypeAppCardZod.merge(
+  z.object({
+    isSunrise: z.boolean(),
```

Comment:
@hariombalhara this can prob. be removed, right?

Reply:
Oh locally removed. Forgot to push. Done now